Formula/jabba.rb

```diff
@@ -20,25 +21,58 @@ class Jabba < Formula
   depends_on "go" => :build
 
   def install
-    ENV["GOPATH"] = buildpath
-    ENV["GO111MODULE"] = "auto"
-    dir = buildpath/"src/github.com/Jabba-Team/jabba"
-    dir.install buildpath.children
-    cd dir do
-      ldflags = "-X main.version=#{version}"
-      system "go", "build", *std_go_args(ldflags: ldflags)
+    ENV["JABBA_GET"] = "false"
+
+    inreplace "Makefile", " sh install.sh", " bash install.sh --skip-rc"
+    inreplace "install.sh" do |s|
+      # Call unaliased rm
+      s.gsub! "  rm -f", "  command rm -f"
+
+      # Homebrew install locations
+      s.gsub! "$JABBA_HOME_TO_EXPORT/bin/jabba", "#{opt_bin}/jabba"
+      s.gsub! "${JABBA_HOME}/bin", bin.to_s
+      s.gsub! "${JABBA_HOME}/jabba.sh", "#{pkgshare}/jabba.sh"
+      s.gsub! "${JABBA_HOME}/jabba.fish", "#{pkgshare}/jabba.fish"
     end
```

Comment:
Could you make an upstream issue or pull request to make these configurable and reference it here?

Reply:
Added at https://github.com/Homebrew/homebrew-core/pull/119957/files#diff-21b44b2361fc470b49d829a6c3d512881cf16415102a6230531a2ee6c56f6294R27